Dockerfile.template

```diff
@@ -97,22 +97,18 @@ RUN set -ex; \
 	extraConfigureArgs=''; \
 	dpkgArch="$(dpkg --print-architecture)"; \
 	case "$dpkgArch" in \
-# with-arch: https://salsa.debian.org/toolchain-team/gcc/-/blob/gcc-11-debian/debian/rules2#L462-502
-# with-float: https://salsa.debian.org/toolchain-team/gcc/-/blob/gcc-11-debian/debian/rules.defs#L502-512
-# with-mode: https://salsa.debian.org/toolchain-team/gcc/-/blob/gcc-11-debian/debian/rules.defs#L480
```

Comment:
Not familiar with what this does?

Reply:
Just updating the links to Debian's packaging of how they build gcc so that we know where we got these flags from.